src/components/inputs/multi-select/multi-select.component.tsx

```diff
@@ -51,7 +51,7 @@ export const OHRIMultiSelect: React.FC<OHRIFormFieldProps> = ({ question, onChan
     }));
 
   const initiallySelectedQuestionItems = [];
-  questionItems.forEach((item) => {
+  questionItems.forEach(item => {
```

Comment:
Usually, I'm all in favour of fore-going React hooks, but from the name here, it seems like both `questionItems` and this could benefit from `useMemo()`, since we only really need to recalculate them if the `question` changes.